specs/vm/instruction_set.md

```diff
@@ -716,6 +733,20 @@ Panic if:
 
 - `$is + imm * 4 > VM_MAX_RAM - 1`
 
+### JNE: Jump if not equal
```

Comment:
Isn't `jne $ra $rb $rc` redundant to (or some optimized version of)

```
eq $rd $ra $rb
eq $re $rd $zero
mul $re $re $rc
add $rd $rd $re
j $rd
```

I understand we have `JNEI` and `JNZI` but I always saw them as special cases because of their immediate values. However, in this case, we have only runtime registers.

Reply:
Haha you're not wrong, but conditional jumps will be most jumps. Definitely worth 1/5th the cost of them!